mentoring/programs/mentoring-lead.md

```diff
@@ -8,10 +8,13 @@ The Mentoring Lead role is created to help upstream Kubernetes contributors to p
  The role lead should:
 
 - Keep a calendar of dates for each program (when to submit our project name, when to submit the project scope, when selecting period is, etc) throughout the   year and
-- Coordinate all submissions and participation in each one
-- Go to sig meetings and slack channels to proactively help with scoping and planning work / job descriptions
-- Help onboard interns to the k8s specific community items like coordinate a hello for them at the k8s monthly community meeting
-- Help off board interns to the k8s specific community items like demo at monthly community meeting
+- Reach out to SIGs to submit proposals/projects so as to ensure healthy number of projects for an internship cohort.
+- Coordinate all submissions and participation in each one, responsible for following up with mentors during evaluation period.
+- Go to sig meetings and slack channels to proactively help with scoping and planning work / job descriptions.
+- Help onboard interns to the k8s specific community items like coordinate a hello for them at the k8s monthly community meeting, 
+- Help off board interns to the k8s specific community items like demo at monthly community meeting.
+- Be a point of contact (unless delegated otherwise) for applicants and mentees, and should respond to queries within a reasonable amount of time.
+- Should try and delegate work among shadows and mentor the shadows along the way.
```

Comment:
@Debanitrkl thanks for the follow-up!

I think the only remaining comment is to add succession planning as an explicit responsibility here - https://github.com/kubernetes/community/pull/6570#discussion_r852565032